scripts/devenv-builder/configure-vm.sh

```diff
@@ -117,17 +149,22 @@ gpgkey=file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-SIG-NFV
 EOF
 fi
 
-if [ $BUILD_AND_INSTALL = true ] ; then
-    sudo dnf localinstall -y ~/microshift/_output/rpmbuild/RPMS/*/*.rpm
+# Set up pull secret for CRI-O regardless if it'll be installed in next step or later from another source
+title "Setting up pull secret for CRI-O"
+sudo mkdir -p /etc/crio/
+sudo cp -f ${OCP_PULL_SECRET} /etc/crio/openshift-pull-secret
+sudo chmod 600 /etc/crio/openshift-pull-secret
```

Comment:
If we want to make the pull secret setup unconditional, let's do it without changing the order: first install RPMs and then the pull secret. This is to adhere to the current docs.

Reply:
I noticed the unconditional call to this code fails in CI. Why do we need it unconditionally?
```
+ echo -e '\E[34mSetting up pull secret for CRI-O\E[00m'
+ sudo mkdir -p /etc/crio/
+ sudo cp -f /etc/crio/openshift-pull-secret /etc/crio/openshift-pull-secret
cp: '/etc/crio/openshift-pull-secret' and '/etc/crio/openshift-pull-secret' are the same file
```

Reply:
I'll swap the order so it's matching what's in the doc (although I wish there was "literal bash" - readable script with embedded docs, because that's a duplication right now).
I want this unconditionally to set up the pull secret without installing RPMs for the CI - cri-o will be pulled in when we install microshift RPMs but that happens after this script